adapter/datadog/logger.go

```diff
@@ -0,0 +1,64 @@
+package datadog
+
+import (
+	"errors"
+	"strings"
+
+	coopLogger "github.com/coopnorge/go-logger"
+)
+
+// Logger is a logging adapter between gopkg.in/DataDog/dd-trace-go.v1/ddtrace
+// an go-logger, do not create this directly, use NewLogger()
+type Logger struct {
+	instance *coopLogger.Logger
+}
+
+// NewLogger creates a new Gorm logger that passes message to go-logger
+//
+// To inject the logger into ddtrace use
+//
+//	package main
+//
+//	import (
+//		datadogLogger "github.com/coopnorge/go-logger/adapter/datadog"
+//
+//		"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+//	)
+//
+//	func main() {
+//		l, err =: gormLogger.NewLogger(gormLogger.WithGlobalLogger())
+//		ddtrace.UseLogger(l)
+//	}
```

Comment:
Sorry, another nitpick: Copy-paste errors in the comments, this is referencing the GOrm-adapter, not the datadog adapter

Reply:
fixed